src/account_facebook/facebook_account_controller.ts

```diff
@@ -513,14 +533,11 @@ export class FacebookAccountController {
 
                         // Check if it's a shared post by looking for external_context in attachments
                         const isSharedPost = post.attachments?.[0]?.data?.[0]?.external_context !== undefined;
```

Comment:
I think the logic for determining whether its a repost or if the post just includes other URLs need to be worked similar to the comment by @micahflee at https://github.com/lockdown-systems/cyd/pull/405#discussion_r1978472981

Reply:
Modified this per https://github.com/lockdown-systems/cyd/pull/405#discussion_r1979791144